VideoLocker/src/main/java/org/edx/mobile/http/HttpStatus.java

```diff
@@ -0,0 +1,288 @@
+package org.edx.mobile.http;
+
+/**
+ * A comprehensive collection of constants for HTTP status codes.
+ */
+public class HttpStatus {
```

Comment:
Do we need to validate the status codes as well? Otherwise an enum would only complicate the comparison, as the status codes we get from the client would need to be converted into the appropriate enum type, while having to handle a null value or an exception in the case where it's not represented in this set.